providers/mailtrap/package.json

```diff
@@ -0,0 +1,82 @@
+{
+  "name": "@novu/mailtrap",
+  "version": "0.20.0-alpha.0",
```

Comment:
provider template generator is "outdated", so I changed to that version manually.